robottelo/ui/subscription.py

```diff
@@ -1,13 +1,21 @@
 """Implements Subscriptions/Manifest handling for the UI"""
 
-from robottelo.helpers import escape_search
 from robottelo.ui.base import Base
 from robottelo.ui.locators import common_locators, locators
+from robottelo.ui.navigator import Navigator
 
 
 class Subscriptions(Base):
```

Comment:
Could you please clarify why search functionality work properly for that entity without `is_katello = True`?